modules/configuration-api/src/main/java/org/apache/ignite/configuration/ConfigurationModule.java

```diff
@@ -97,7 +97,10 @@ default Collection<Class<?>> polymorphicSchemaExtensions() {
 
     /**
      * Patches the provided configuration with dynamic default values. This method is called
-     * only for cluster-wide configuration on cluster initialization.
+     * <ul>
+     *     <li>for cluster-wide configuration on cluster initialization.</li>
+     *     <li>for node-local configuration on configuration read.</li>
```

Comment:
What do you mean by "configuration read"? Is it the first read? Or is it on configuration instantiation? Or is it patched on every read?

Reply:
Changed